autonomy/cli/analyse.py

```diff
@@ -97,8 +97,23 @@ def check_abci_app_specs(
 ) -> None:
     """Check ABCI app specs."""
 
+    # The command expects 'packages_dir' to be named 'packages', so to make import statements to be compatible with
+    # the standard Python import machinery. The directory can be outside the working directory from which the command
+    # is executed.
```

Comment:
I'd have put those comments in the function docstring such that Click could show them in the command help message. However, Darglint complains if the argument documentation is not present; on the other hand, if we added the explanation of the documentation, this would appear in the command helper message, which does not make much sense.